app/genesis-tool/Run.hs

```diff
@@ -144,6 +130,70 @@ runCommand
   where
     fromKMO = decideKeyMaterialOps fromVer
 
+runCommand
+  kmo
+  (DumpHardcodedGenesis
+    outDir)
+  =
+  dumpGenesis kmo outDir
+  (configGenesisData Dummy.dummyConfig)
+  $ fromMaybe (error "Hardcoded genesis lacks secrets.") (configGeneratedSecrets Dummy.dummyConfig)  
+
+runCommand
+  KeyMaterialOps{..}
+  (PrintGenesisHash
+    secretPath)
+  =
+  putStrLn =<< F.format CCr.hashHexF . unGenesisHash . snd . either (error . show) id <$> (runExceptT $ readGenesisData secretPath)
+  where
+
+runCommand
+  KeyMaterialOps{..}
+  (PrintSigningKeyAddress
+    networkMagic
+    secretPath)
+  =
+  putStrLn =<< T.unpack . prettyAddress . CC.makeVerKeyAddress networkMagic . CCr.toVerification . kmoDeserialiseDelegateKey <$> LB.readFile secretPath
```

Comment:
I wouldn't force it this much, but it's a matter of style, so fine.

Reply:
This line is too long.  If you fit in 80 characters, those with smaller monitors (laptos with high resolution) or bigger monitors can fit splits of 2 (or more). This is really handy when working with code.